runtime/CMakeLists.txt

```diff
@@ -325,6 +323,67 @@ add_subdirectory(gc_glue_java)
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 add_subdirectory("${J9VM_OMR_DIR}" omr)
 
+# Configure our DDR set
+if(OMR_DDR)
+	make_ddr_set(j9ddr)
+	ddr_set_add_targets(j9ddr omrddr)
+	set_target_properties(j9ddr PROPERTIES
+		BLACKLIST "${CMAKE_CURRENT_SOURCE_DIR}/ddr/blacklist"
+		OVERRIDES_FILE "${CMAKE_CURRENT_SOURCE_DIR}/ddr/overrides"
+		BLOB "${CMAKE_CURRENT_BINARY_DIR}/j9ddr.dat"
+		SUPERSET "${CMAKE_BINARY_DIR}/superset.dat"
+	)
+
+	ddr_add_headers(j9ddr
+		oti/bcverify.h
```

Comment:
Please explain why these particular header files are listed explicitly rather than using `GLOB_HEADERS` as is done elsewhere.

If there *is* a good reason for enumerating them, please put them in order (perhaps in groups with a comment explaining each group).

Reply:
The idea is that the headers as associated with certain components. Ie `oti/j9port.h` is associated with the port library component, and if you disable the port library you wont get those headers.  (although the port library is a bad example, because it can't really be disabled)

Reply:
What you say about components makes sense, but aren't they organized in separate directories and so have their own `CMakeLists.txt` files?

I still don't understand why this file lists only certain specific header files.

Reply:
The ones in this list, are onews that are not easy to classify / havent been classified under a more specific target. 

I believe the headers here are all the ones with `@ddr_options` or `@ddr_namespace`

Reply:
In what sense are they 'not easy'? Why can't we 'glob *.h *.hpp'? Even those without `@ddr_options` or `@ddr_namespace` can be interesting.
The problems will begin when someone introduces new header files and forgets to include them in this list; then we will have a harder time diagnosing problems involving the new types in those header files.

Reply:
The thinking was that if the port library explicityly added `oti/j9port.h` and we also globbed it, we would be parsing the file twice. Not only that but we would still be parsing it even if we had the port library disabled.
However my thinking is that when we explicitly add a file like that to a target, we just remove it from the list of globbed headers. eg

port/CMakeLists.txt
```
....
ddr_add_headers(j9prt
    ${j9vm_SOURCE_DIR}/oti/j9port.h
)
```

CMakeLists.txt:
```
file(GLOB headers RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
	include/*.h
	include/*.hpp
	oti/*.h
	oti/*.hpp
)
list(REMOVE_ITEMS headers
	oti/j9port.h
)

ddr_add_headers(j9ddr
	${CMAKE_CURRENT_BINARY_DIR}/j9cfg.h
	${CMAKE_CURRENT_BINARY_DIR}/j9vmconstantpool.h
	${headers}
)
```

Reply:
I think our cmake system ought to handle listing a header file more than once and just do the right thing (process it once). Even if that means redundant make rules are generated, it should not be a problem because they should all yield the same result and only one should notice that work needs to be done.